playground.sql

```diff
@@ -1,3 +1,12 @@
 \c nc_news_test;
 
-SELECT * FROM topics;
\ No newline at end of file
+SELECT * FROM articles ;
+--SELECT articles.author,articles.title,articles.article_id,articles.topic,articles.created_at,articles.votes,articles.article_img_urlFROM articles,comments.article_id FROM articles JOIN articles.article_id=comments.article_id;
```

Comment:
Make sure to add any files such as this and the text file to your git ignore file and delete from git hub. 

Also - make sure to update your end points JSON after you add each end point ✨